Source/GoBot/GoBot.swift

```diff
@@ -37,7 +37,13 @@ class GoBot: Bot {
     var identity: Identity? { return self._identity }
 
     private let queue: DispatchQueue
-
+    var follows: [FeedIdentifier] = []
+    var withoutPubs: [FeedIdentifier] = []
+    var followedBy: [FeedIdentifier] = []
+    var friends: [Identity] = []
+    var blocks: [FeedIdentifier] = []
+    var anyBlocks: Bool = false
```

Comment:
Are we reading these variables? I only see writes

Reply:
self.followedBy = withoutPubs

We use these as caches for your personal list of people you follow / contacts / are followed by / friends.